### PR TITLE
Multi-GSIM management in scenario calculators

### PR DESCRIPTION
**Pull request by micheles**
See https://bugs.launchpad.net/oq-engine/+bug/1400042. The tests are green:
https://ci.openquake.org/job/zdevel_oq-risklib/517/

### CODE REVIEW

File: openquake/commonlib/export/hazard.py [hunk 1]
```diff
@@ -191,44 +193,44 @@ def __iter__(self):
...
+                   gsim_path):
     """
     :param key: output_type and export_type
     :param export_dir: the directory where to export
     :param fname: name of the exported file
     :param sitecol: site collection
     :rupture_tags: a list of rupture tags
-    :gmfs: a dictionary of ground motion fields keyed by IMT
+    :gmfs: a matrix of ground motion fields of shape (R, N)
     """
```
Comment (ptormene) [inline openquake/commonlib/export/hazard.py]: Please add :param gsim_path:

File: openquake/commonlib/export/hazard.py [hunk 2]
```diff
@@ -191,44 +193,44 @@ def __iter__(self):
...
+                   gsim_path=None):
     """
     :param key: output_type and export_type
     :param export_dir: the directory where to export
     :param fname: name of the exported file
     :param sites: a filtered site collection
     :rupture_tags: a list of rupture tags
-    :gmfs: a dictionary of ground motion fields keyed by IMT
+    :gmfs: a list of ground motion fields
     """
```
Comment (ptormene) [inline openquake/commonlib/export/hazard.py]: Same as above

**Comment by ptormene** [comment]
Besides the docstrings, LGTM

**Comment by micheles** [comment]
I have fixed the docstrings

**Pull request merged**